chidioguejiofor/ojumo-bend
# Sets up backend repo

#### What does this PR achieve?
- install express and other dependencies
- adds setup for Sequelize
- creates folder structure for the backend
- sets up the route structure and first index route
- sets up testing infrastructure using `mocha` and `chai`

#### Any background context?
- We intend to write tests for backend features but not frontend

#### Manual Testing Steps
- Start the server via: `npm run dev`
- Make a get request to `\api` 
- You should see a sample message
 
#### Screenshots:

#### Relevant Trello Tickets:
- [Backend Setup](https://trello.com/c/l864ioiS/1-create-and-setup-backend)
